src/components/ConnectExchangeView/ExchangeAccountConnect/ExchangeAccountConnect.js

```diff
@@ -131,6 +132,46 @@ const ExchangeAccountConnect = () => {
     );
   }
 
+  /**
+   * Function to return the url for selected exchange.
+   *
+   * @param {string} name Name of selected exchange.
+   * @returns {JSX.Element} Url of the selected exchange.
+   */
+  const exchangeUrl = (name) => {
+    switch (name) {
+      case "binance":
+        return (
+          <a className="exchangeLink" href={binanceUrl} rel="noopener noreferrer" target="_blank">
```

Comment:
Can we write the anchor element only once, and use the switch to return the url string only?
```       
<a className="exchangeLink" href={url} rel="noopener noreferrer" target="_blank">
  {url}
</a>
```

Reply:
@cwagner22  yeah, agreed, Thanks!